jobs/build/ocp4/Jenkinsfile

```diff
@@ -148,15 +148,38 @@ node {
                 }
 
                 stage("update dist-git") { build.stageUpdateDistgit() }
-                stage("build images") { build.stageBuildImages() }
+                stage("build images") {
+                    build.stageBuildImages()
```

Comment:
The "alert on embargoes" part here should really be a separate commit (maybe a separate PR) from the "deal with image failures" sections later.

Also could the new logic go inside `stageBuildImages`? I prefer to keep this Jenkinsfile high-level.